notes.dart

```diff
@@ -0,0 +1,28 @@
+/*
+
+CHEAT CODE : Add the User Freezed form the old project 
+
+App State Setup
```

Comment:
@verywellced what's this all about? do still need this code change here? 🙂

Reply:
hehehe thats my notes sir, will transfer it after the  class is that ok?